src/profile-logic/stack-timing.js

```diff
@@ -66,7 +66,7 @@ type LastSeen = {
  * Build a StackTimingByDepth table from a given thread.
  */
 export function getStackTimingByDepth(
-  thread: Thread,
```

Comment:
To compute the stack timings we weren't using the "samples like" table: so we were computing a stack timing from the samples, and using it for the memory information... and BOOOM.

Reply:
Boom indeed.